webviews/components/sidebar.tsx

```diff
@@ -144,44 +144,71 @@ export default function Sidebar({ reviewers, labels, hasWritePermission, isIssue
 					) : null}
 				</div>
 				{milestone ? (
-					<div className="section-item label">
-						{milestone.title}
-						{hasWritePermission ? (
-							<>
-								{nbsp}
-								<button
-									className="push-right remove-item"
-									onClick={async () => {
-										await removeMilestone();
-										updatePR({ milestone: null });
-									}}
-								>
-									{deleteIcon}️
-								</button>
-								{nbsp}
-							</>
-						) : null}
-					</div>
+					<Milestone key={milestone.title} {...milestone} canDelete={hasWritePermission} />
 				) : (
 					<div className="section-placeholder">No milestone</div>
 				)}
 			</div>
-		</div>
+		</div >
 	);
 }
 
 function Label(label: ILabel & { canDelete: boolean }) {
-	const { name, canDelete } = label;
-	const { removeLabel } = useContext(PullRequestContext);
+	const { name, canDelete, color } = label;
+	const { removeLabel, pr } = useContext(PullRequestContext);
+	const labelColor = gitHubLabelColor(color, pr.isDarkTheme, false);
 	return (
 		<div
 			className="section-item label"
+			style={{
+				backgroundColor: labelColor.backgroundColor,
+				color: labelColor.textColor,
+				border: `1px solid ${labelColor.borderColor}`
+			}}
 		>
 			{name}
 			{canDelete ? (
 				<>
 					{nbsp}
-					<button className="push-right remove-item" onClick={() => removeLabel(name)}>
+					<button className="push-right remove-item"
+						onClick={() => removeLabel(name)}
+						style={{ stroke: labelColor.textColor }}
+					>
+						{deleteIcon}️
+					</button>
+					{nbsp}
+				</>
+			) : null}
+		</div>
+	);
+}
+
+function Milestone(milestone: IMilestone & { canDelete: boolean }) {
+	const { removeMilestone, updatePR, pr } = useContext(PullRequestContext);
+	const backgroundBadgeColor = getComputedStyle(document.documentElement).getPropertyValue('--vscode-badge-foreground');
+	const labelColor = gitHubLabelColor(backgroundBadgeColor, pr.isDarkTheme, false);
+	const { canDelete, title } = milestone;
+	return (
+		<div
+			className="section-item label"
+			style={{
+				backgroundColor: labelColor.backgroundColor,
+				color: labelColor.textColor,
+				border: `1px solid ${labelColor.borderColor}`
```

Comment:
Since the 1px solid border is constant can it be moved into
https://github.com/microsoft/vscode-pull-request-github/blob/674a0d7363c0e7ec1cd321a1de9e4e90be555045/webviews/editorWebview/index.css#L463-L470
?